<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="rb-ssuri" failures="1" tests="5" name="TestP" time="56.823" errors="0" timestamp="2024-10-22T11:23:46 IST">
  <testcase classname="script.Demo1" name="testA" time="0.083"/>
  <testcase classname="script.Demo2" name="testB" time="0.015">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at script.Demo2.testB(Demo2.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- testB -->
  <testcase classname="script.Demo4" name="createUser" time="0.001"/>
  <testcase classname="script.DemoX" name="createUser" time="0.0"/>
  <testcase classname="script.DemoX" name="deleteUser" time="0.002"/>
  <testcase classname="generic.BaseTest" name="@BeforeSuite initReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite initReport -->
  <testcase classname="generic.BaseTest" name="@AfterSuite generateReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite generateReport -->
</testsuite> <!-- TestP -->
